CHANGELOG.md

```diff
@@ -7,22 +7,23 @@ This changelog is a work in progress and may contain notes for versions which ha
 ### Breaking changes 🛠
 
 - As a result of implementing custom order filters, some of the code Mesh uses under the hood to share orders with peers has changed. As a result this version of Mesh cannot share orders with any older versions and vice versa ([#630](https://github.com/0xProject/0x-mesh/pull/630)).
+- Rename `RPC_ADDR` to `WS_RPC_ADDR` since we now support both WS and HTTP JSON-RPC endpoints. ([#658](https://github.com/0xProject/0x-mesh/pull/658))
 
 ### Features ✅
 
 - Implemented custom order filters, which allow users to filter out all but the orders they care about. When a custom order filter is specified, Mesh will only send and receive orders that pass the filter. ([#630](https://github.com/0xProject/0x-mesh/pull/630)).
 - Developers can now override the contract addresses for any testnet using the `CUSTOM_CONTRACT_ADDRESSES` env config ([#640](https://github.com/0xProject/0x-mesh/pull/640)).
 - Added `getOrdersForPageAsync` method to `@0x/mesh-rpc-client` WS client interface so that clients can paginate through the retrieved orders themselves ([#642](https://github.com/0xProject/0x-mesh/pull/642)).
-- Added `getStatsAsync` to the `@0x/mesh-browser` package. ([#654](https://github.com/0xProject/0x-mesh/pull/654)).
+- Added `getStatsAsync` to the `@0x/mesh-browser` package. This exposes the `getStats` method to browser users. ([#654](https://github.com/0xProject/0x-mesh/pull/654)).
+- Add support for calling JSON-RPC methods over HTTP (env config `HTTP_RPC_ADDR` defaults to `localhost:60556`). ([#658](https://github.com/0xProject/0x-mesh/pull/658))
 - Added `getOrdersAsync` and `getOrdersForPageAsync` to the `@0x/mesh-browser` package. ([#655](https://github.com/0xProject/0x-mesh/pull/655)).
 - Added support for passing in your own Web3 provider when using the `@0x/mesh-browser` package. ([#665](https://github.com/0xProject/0x-mesh/pull/665)).
 
-
 ## v8.1.2
 
 ### Bug fixes 🐞
 
-- Update DevUtils contract addresses to version that fixes issues for Augur. ([#662](https://github.com/0xProject/0x-mesh/pull/662)).
+- Update DevUtils contract to version that removed maker transfer simulation. ([#662](https://github.com/0xProject/0x-mesh/pull/662)).
```

Comment:
This should have been changed in @jalextowle PR but since it was missed, I'll update it now.